corehq/apps/registration/utils.py

```diff
@@ -131,17 +132,39 @@ def request_new_domain(request, form, is_new_user=True):
 
     if is_new_user:
         dom_req.save()
-        send_domain_registration_email(request.user.email,
-                                       dom_req.domain,
-                                       dom_req.activation_guid,
-                                       request.user.get_full_name(),
-                                       request.user.first_name)
+        if settings.IS_SAAS_ENVIRONMENT:
+            load_appcues_template_apps.apply_async(
+                (new_domain.name, current_user.username),
+                link=send_domain_registration_email(
+                    request.user.email,
+                    dom_req.domain,
+                    dom_req.activation_guid,
+                    request.user.get_full_name(),
+                    request.user.first_name
+                )
+            )
+        else:
+            send_domain_registration_email(request.user.email,
+                                           dom_req.domain,
+                                           dom_req.activation_guid,
+                                           request.user.get_full_name(),
+                                           request.user.first_name)
     send_new_request_update_email(request.user, get_ip(request), new_domain.name, is_new_user=is_new_user)
 
     send_hubspot_form(HUBSPOT_CREATED_NEW_PROJECT_SPACE_FORM_ID, request)
     return new_domain.name
 
 
+APPCUES_TEMPLATE_SLUGS = ['health', 'aggriculture', 'wash']
+
+
+@task(serializer='pickle', queue='background_queue')
```

Comment:
Can the `serializer` be removed? `serializer=pickle` is there on other tasks because the default from 3.x to 4.x changes from pickle to json. we changed the default for hq to be json and all new tasks should use that when possible